public/assets/css/contact.css

```diff
@@ -0,0 +1,98 @@
+
+@font-face {
+    font-family: 'IstokWebRegular';
+    src: url('../fonts/IstokWeb-Regular.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'IstokWebBold';
+    src: url('../fonts/IstokWeb-Bold.ttf') format('truetype');
+    font-weight: bold;
+    font-style: normal;
+    font-display: swap;
+}
+
+:root {
+    --color-primary: #224151;
+    --color-secondary: #F1EFDA;
+    --color-text: #000;
+    --font-primary-regular: IstokWebRegular;
+    --font-primary-bold: IstokWebBold;
+    --font-size-cart-header: 18px;
+    --font-size-cart-date: 16px;
+    --font-size-cart-footer: 1rem;
+    --padding-cart: 20px;
+    --cart-header-height: 54px;
+}
+
+
+
+input[type=text], textarea, input[type=email] {
```

Comment:
if this behavior works for all input with types selected, put it inside the global css file